lib/report.js

```diff
@@ -188,43 +202,96 @@ class Report {
     }
 
     if (this.all) {
-      const emptyReports = []
+      const emptyReports = this._includeUncoveredFiles(fileIndex)
       v8ProcessCovs.unshift({
         result: emptyReports
       })
-      const workingDirs = this.src
-      const { extension } = this.exclude
-      for (const workingDir of workingDirs) {
-        this.exclude.globSync(workingDir).forEach((f) => {
-          const fullPath = resolve(workingDir, f)
-          if (!fileIndex.has(fullPath)) {
-            const ext = extname(fullPath)
-            if (extension.includes(ext)) {
-              const stat = statSync(fullPath)
-              const sourceMap = getSourceMapFromFile(fullPath)
-              if (sourceMap) {
-                this.sourceMapCache[pathToFileURL(fullPath)] = { data: sourceMap }
-              }
-              emptyReports.push({
-                scriptId: 0,
-                url: resolve(fullPath),
-                functions: [{
-                  functionName: '(empty-report)',
-                  ranges: [{
-                    startOffset: 0,
-                    endOffset: stat.size,
-                    count: 0
-                  }],
-                  isBlockCoverage: true
-                }]
-              })
-            }
+    }
+
+    return mergeProcessCovs(v8ProcessCovs)
+  }
+
+  async _getMergedProcessCovAsync () {
```

Comment:
nit: perhaps add a jsdoc string above here, as we have with the sync version of the method.